test/draw/RasterRgbaTest.ooc

```diff
@@ -16,6 +16,15 @@ RasterRgbaTest: class extends Fixture {
 	sourceFlower := "test/draw/input/Flower.png"
 	init: func {
 		super("RasterRgbaTest")
+		this add("Test file not found", func {
+			try {
+				image := RasterRgba open("nonExistingFile")
+				expect(false)
```

Comment:
>  I don't want the critical part of the test to be optimized away

I don't think gcc will completely remove a call to external function with possible side effects even with -O3, but I think you are right, we should remove this warning.